src/generate.rs

```diff
@@ -176,7 +179,16 @@ pub fn implement(field: &Field, params: &GenParams) -> TokenStream2 {
                     }
                 }
             }
-            GenMode::GetCopy => {
+            GetClone => {
+                quote! {
+                    #(#doc)*
+                    #[inline(always)]
+                    #visibility fn #fn_name(&self) -> #ty {
+                        self.#field_name.clone()
```

Comment:
My understanding is that this will not do what you want it to do. The reason I asked in the issue whether you intended to have a special case for `Arc` is that the `Arc` clone method can only be invoked via `Arc::clone()` not `self.clone()`. As I understand it, this will attempt to clone the data structure contained inside of the `Arc`.